plugins/uberdev/commands/solve.md

```
@@ -4,6 +4,45 @@ argument-hint: "<issue-number> [--trivial|--small|--full] [--auto] [--terminal=c
 allowed-tools: ["Bash", "Read", "Task"]
 ---
 
+<!--
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)